src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -736,58 +736,44 @@ void UpdateDimensionProperties()
 			}
 			else if (Content is FrameworkElement fe)
 			{
-				double CalculateExtent(double @explicit, double actual, double desired, bool isStretchAlign, double margin)
+				ExtentHeight = CalculateExtent(this, fe, isHorizontal: false);
+				ExtentWidth = CalculateExtent(this, fe, isHorizontal: true);
+
+				static double CalculateExtent(ScrollViewer sv, FrameworkElement fe, bool isHorizontal)
```

Comment:
is it worth tagging the two with aggressive inlining?

Reply:
Not sure it's possible with nested functions and my personal preference would be to keep them here close to their usages. It's not a such hot-path (only warm :P) so I think it's fine.